Tests/SonarQube.Common.UnitTests/ProcessRunnerTests.cs

```diff
@@ -276,14 +278,17 @@ public void ProcRunner_DoNotLogSensitiveData()
                 "/d:sonar.login=secret data login",
                 "/d:sonar.jdbc.password=secret data db password",
                 "/d:sonar.jdbc.username=secret data db user name",
+                "/d:sonar.vbnet.license.secured=mypass",
 
                 // Sensitive args - different cases -> exclude to be on the safe side
                 "/d:SONAR.jdbc.password=secret data db password upper",
                 "/d:sonar.PASSWORD=secret data password upper",
+                "/d:sonar.license.SECURED=mypass",
 
                 // Sensitive args - parameter format is slightly incorrect -> exclude to be on the safe side
                 "/dsonar.login =secret data key typo",
-                "sonar.password=secret data password typo"
+                "sonar.password=secret data password typo",
+                "/d:sonar.license.secured = mypass",
```

Comment:
The check below expects all of the secret data to include the word "secret" in the value. Same applies to line 286.